ap.md

```diff
@@ -0,0 +1,236 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+# Build Your Own Internet
+<div class="large-8 columns">
+
+Along with connecting to an existing Wifi network, Tessel 2 can also act as a portable router and emit a custom Wifi network called an access point. We can use this functionality to connect directly with our Tessel through a web browser. The advantage of doing this through Tessel's access point instead of through a local Wifi network is reduced latency of requests to the Tessel since they won't be making a detour through a different router. This also means there is no need for a local network for connecting to Tessel through the web. By the end of this article, we'll be able to control the Tessel LEDs through a web app served by the Tessel.  
+
+In your command line, make a folder for your Tessel code, then initialize a Tessel project in that folder by running each of the following commands in the terminal:
+
+`mkdir tessel-router`
+
+`cd tessel-router`
+
+`t2 init`
+
+Great! Now you're set up to run code on Tessel. Your "tessel-router" folder now contains a "package.json" with some metadata Node uses for your project, and a file called "index.js".
```

Comment:
Better to match the form of the other tutorials & reduce this section (see e.g. the tweet page). We explained the `init` command when we first used it on the blinky page.